src/platform/unix.rs

```diff
@@ -470,62 +470,52 @@ impl MonitorHandleExtUnix for MonitorHandle {
 }
 
 /// A theme for a Wayland's client side decorations.
-///
-/// The colors are in ARGB format.
+#[cfg(feature = "wayland")]
 pub trait Theme: Send + 'static {
-    /// Primary color of the scheme.
-    fn primary_color(&self, window_active: bool) -> [u8; 4];
+    /// Title bar color.
+    fn titlebar_element_color(&self, element: TitleBarElement, window_active: bool) -> ARGBColor;
```

Comment:
```suggestion
    fn element_color(&self, element: Element, window_active: bool) -> ARGBColor;
```

It's all about the title bar, it's the title bar theme after all.